src/components/menuBar/js/menuItemDirective.js

```diff
@@ -17,9 +17,12 @@ function MenuItemDirective() {
         var buttonEl = angular.element('<md-button type="button"></md-button>');
             buttonEl.html(text);
             buttonEl.attr('tabindex', '0');
-
         templateEl.html('');
-        templateEl.append(angular.element('<md-icon md-svg-icon="check"></md-icon>'));
+        if(templateAttrs.icontype=="svg"){
+          var iconElement = angular.element('<md-icon md-svg-icon="check"></md-icon>');
+        }
+        var iconElement = angular.element('<md-icon>check</md-icon>');
```

Comment:
That won't work correctly, you will always overwrite the `iconElement`. (when using `icontype="svg"`)